manim/animation/growing.py

```diff
@@ -8,21 +8,30 @@
     "SpinInFromNothing",
 ]
 
+import typing
+
+import numpy as np
 
 from ..animation.transform import Transform
 from ..constants import PI
 
+if typing.TYPE_CHECKING:
+    from ..mobject.geometry import Arrow
+    from ..mobject.mobject import Mobject
+
 
 class GrowFromPoint(Transform):
-    def __init__(self, mobject, point, point_color=None, **kwargs):
+    def __init__(
+        self, mobject: "Mobject", point: np.ndarray, point_color: str = None, **kwargs
```

Comment:
why is Mobject a sting here?

Reply:
That's how it's done when thing's aren't defined. Here, `Mobject` isn't imported so it is in string.
See https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
Ah, nice, thanks a lot, I did not know that.

Reply:
Just realized that here https://www.python.org/dev/peps/pep-0484/#forward-references they use single quotes: ' ' and here they are double quotes: " " . But I guess that won't become a problem?

Reply:
Hmmmmm. You know we have something called **BLACK** , which doesn't like single quotes?

Reply:
okay, then we can stay like this.